### PR TITLE
feature/jenova: include proto/port in suggested policies

### PR DESCRIPTION
**Pull request by malkhamis-panw**
For more information, please read the description in [this MR](https://git.scm.prismacloud.io/prismacloud/pcn/backend/-/merge_requests/1472)

### CODE REVIEW

**Comment by malkhamis-panw** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "feature-jeonva-suggested-policies-proto-port",
    "component": "gaia",
    "pr-id": "747",
    "commit-sha": "6649fd6e03c4a4004f0b483519766ab172716c23",
    "pipeline": "master"
  },
  {
    "project": "feature-jeonva-suggested-policies-proto-port",
    "component": "backend",
    "pr-id": "1472",
    "commit-sha": "a2871b6d4f3c6e7fed6a507b138339f866282130",
    "pipeline": "master"
  }
]
```

**Comment by malkhamis-panw** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "feature-jeonva-suggested-policies-proto-port",
    "component": "gaia",
    "pr-id": "747",
    "commit-sha": "6649fd6e03c4a4004f0b483519766ab172716c23",
    "pipeline": "master"
  },
  {
    "project": "feature-jeonva-suggested-policies-proto-port",
    "component": "backend",
    "pr-id": "1472",
    "commit-sha": "3d588800c85b0972b9711d6bd2731ea19d8e28d4",
    "pipeline": "master"
  }
]
```

**Comment by malkhamis-panw** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "feature-jeonva-suggested-policies-proto-port",
    "component": "gaia",
    "pr-id": "747",
    "commit-sha": "6649fd6e03c4a4004f0b483519766ab172716c23",
    "pipeline": "master"
  },
  {
    "project": "feature-jeonva-suggested-policies-proto-port",
    "component": "backend",
    "pr-id": "1472",
    "commit-sha": "99a992d16129eb3d2a5cad2fba013cfc0ce66a74",
    "pipeline": "master"
  }
]
```

**Comment by malkhamis-panw** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "feature-jeonva-suggested-policies-proto-port",
    "component": "gaia",
    "pr-id": "747",
    "commit-sha": "6649fd6e03c4a4004f0b483519766ab172716c23",
    "pipeline": "master"
  },
  {
    "project": "feature-jeonva-suggested-policies-proto-port",
    "component": "backend",
    "pr-id": "1472",
    "commit-sha": "a94632ee489ba0d63227f7814b2ec49685abd59b",
    "pipeline": "master"
  }
]
```

File: specs/+graphedgeflowdetails.spec [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  entity_name: GraphEdgeFlowDetails
+  package: meteor
+  group: visualization/depmaps
+  description: Contains more specific information about a particular flow between two nodes.
+  detached: true
+
+# Attributes
+attributes:
+  v1:
+  - name: allowed
```
Comment (primalmotion) [inline specs/+graphedgeflowdetails.spec:14]: We usually use an enum here. In either way the terminology is `accept(ed)`/`reject(ed)`

File: specs/graphedge.spec [hunk 2]
```diff
@@ -106,6 +106,13 @@ attributes:
     - Node
     - RemoteController
 
+  - name: details
+    description: Contains more flow details grouped by their destination protocol/ports.
+    type: external
```
Comment (primalmotion) [inline specs/graphedge.spec:111]: IIRC we have a type `refMap` that would prevent the need for a type mapping

**Review by primalmotion — CHANGES_REQUESTED** [review]
Also, I'm not sure why all the line limits changed here. you need to run rego format

**Comment by malkhamis-panw** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "feature-jeonva-suggested-policies-proto-port",
    "component": "gaia",
    "pr-id": "747",
    "commit-sha": "6649fd6e03c4a4004f0b483519766ab172716c23",
    "pipeline": "master"
  },
  {
    "project": "feature-jeonva-suggested-policies-proto-port",
    "component": "backend",
    "pr-id": "1472",
    "commit-sha": "259cbf116ee207e0a53543d68602b8f832a3ab22",
    "pipeline": "master"
  }
]
```

File: specs/graphedge.spec (hunk 2 above)
Comment (malkhamis-panw) [reply specs/graphedge.spec]: done

File: specs/+graphedgeflowdetails.spec (hunk 1 above)
Comment (malkhamis-panw) [reply specs/+graphedgeflowdetails.spec]: I kept it boolean for simplicity and inline with other `acceptedXX` fields in the parent spec. Changed it to `Accepted`

**Comment by malkhamis-panw** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "feature-jeonva-suggested-policies-proto-port",
    "component": "gaia",
    "pr-id": "747",
    "commit-sha": "e99cacf23fd94d05d4cf744e4f4bc1216fd49bce",
    "pipeline": "master"
  },
  {
    "project": "feature-jeonva-suggested-policies-proto-port",
    "component": "backend",
    "pr-id": "1472",
    "commit-sha": "6261dcd7d63b2b6bc23f92e4959ba98ce012cf30",
    "pipeline": "master"
  }
]
```

**Review by primalmotion — APPROVED** [review]

**Comment by malkhamis-panw** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "feature-jeonva-suggested-policies-proto-port",
    "component": "backend",
    "pr-id": "1472",
    "commit-sha": "a8a166c4d57ee3715d4aff891472667f092b262d",
    "pipeline": "master"
  },
  {
    "project": "feature-jeonva-suggested-policies-proto-port",
    "component": "gaia",
    "pr-id": "747",
    "commit-sha": "e99cacf23fd94d05d4cf744e4f4bc1216fd49bce",
    "pipeline": "master"
  }
]
```

**Comment by malkhamis-panw** [comment]
FT is failing due to a known issue in enforcer per QA team. Suggestion was to force-merge MR for the time being

**Pull request merged**